manager/allocator/network.go

```diff
@@ -857,13 +857,25 @@ func (a *Allocator) doTaskAlloc(ctx context.Context, ev events.Event) {
 	log.G(ctx).Debugf("task %v was marked pending allocation", t.ID)
 }
 
+func nodeIsWindows(node *api.Node) bool {
+	return node != nil &&
+		node.Description != nil &&
+		node.Description.Platform != nil &&
+		node.Description.Platform.OS == "windows"
+
+}
+
 func (a *Allocator) allocateNode(ctx context.Context, node *api.Node, existingAddressesOnly bool, networks []*api.Network) bool {
 	var allocated bool
 
 	nc := a.netCtx
 
 	for _, network := range networks {
 
+		if !nodeIsWindows(node) && !network.Spec.Ingress {
```

Comment:
Please add a comment for this change. thx!

Reply:
Will do.